src/main/scala/org/broadinstitute/dsde/firecloud/EntityService.scala

```diff
@@ -227,8 +227,22 @@ class EntityService(rawlsDAO: RawlsDAO, importServiceDAO: ImportServiceDAO, cwds
     val insertedObject = googleServicesDAO.writeObjectAsRawlsSA(bucketToWrite, fileToWrite, dataBytes)
     val gcsPath = s"gs://${insertedObject.bucketName.value}/${insertedObject.objectName.value}"
 
-    val importRequest = AsyncImportRequest(gcsPath, FILETYPE_RAWLS)
-    importServiceDAO.importJob(workspaceNamespace, workspaceName, importRequest, isUpsert)(userInfo)
+    val importRequest = AsyncImportRequest(gcsPath, FILETYPE_RAWLS, Some(ImportOptions(None, Some(isUpsert))))
+
+    if (cwdsDAO.isEnabled && cwdsDAO.getSupportedFormats.contains(importRequest.filetype.toLowerCase)) {
```

Comment:
Is there a need to check here if Rawls JSON is supported? It's hard-coded to be.

```suggestion
    if (cwdsDAO.isEnabled) {
```

Reply:
Somewhere in there it's getting the info from orch's config, as I discovered by running it in my bee without updating the config.  If we leave this check in here, then I can merge this PR before updating the conf and before cWDS is actually able to take these types of requests without any problem.

Reply:
🤦 I was looking at a test.